discord_bot/message_sync.py

```
@@ -79,11 +89,15 @@ async def sync_all_channels(channels, sync_limit: int = 200):
     failed = 0
     
     for channel in channels:
+        channel_name = getattr(channel, 'name', 'DM')
         try:
             await sync_recent_messages(channel, sync_limit=sync_limit)
             synced += 1
         except Exception as e:
-            logger.error(f"[Sync] Failed to sync channel {channel.id}: {e}")
+            logger.error(
```

Ruff (0.15.2)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)